apps/github-pages/src/components/Nav.astro

```diff
@@ -0,0 +1,63 @@
+---
+const pathname = new URL(Astro.request.url).pathname;
+---
+
+<nav>
+	<a class={pathname === '/' ? 'active' : ''} href="/">CSNX</a>
+	<a class={pathname === '/cmp' ? 'active' : ''} href="/cmp-test-page"
```

Comment:
Is the URL `/cmp` or [`/cmp-test-page`](https://guardian.github.io/csnx/cmp-test-page)?

Reply:
https://github.com/guardian/csnx/pull/1250